### DISCUSSION TITLE
Dragon Ball FighterZ

### DISCUSSION THREAD

**Discussion by supposede** [skyline-emu/skyline#433] [Game Requests]
Very cool and popular game, game play doesn't look too resource demanding but not sure.

**Comment by ASHXSTROM** [comment]
Please make compatible for dragon ball fighter z

**Comment by Pervasive** [comment]
Yes db fighterZ is a 2.5 d game it might work in couple of months

**Comment by MTCypher** [comment]
Yes pls
